artworkSubmitter.js

```diff
@@ -0,0 +1,530 @@
+	182: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/e/e/a/eeaa5a7e43aa336283f05aba426eb74d45aad231.png", "Alcohol", "radical"],
+	181: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/1/8/2/18234a6303223119ee8bb3e885365ad7d5541590.png", "Beans", "radical"],
+	219: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/d/d/e/ddea2b3fc1e72575b85b01c780d4b19e858e8774.png", "Bundle", "radical"],
+	8774: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/2/d/a/2dab43c216ccee57470a860dffc2241c46921185.png", "Chinese", "radical"],
+	186: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/9/a/4/9a42097a9828a9df43c917dd91a69fe3d9e8524e.png", "Dynamite", "radical"],
+	275: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/5/2/6/526c304dec9bb28e328e32e28525df0345750c5a.png", "Fault", "radical"],
+	184: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/c/4/7/c478d4c2a52d890c7dfab63dcdac5adbf74cbdf0.png", "Geoduck", "radical"],
+	180: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/3/d/9/3d9df120e92a14d43fe515e8d4ed9e6a4fe012b7.png", "Kiss", "radical"],
+	183: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/f/5/c/f5c3d110b37bbb7edd343218126d71a99c9ed5fe.png", "Landslide", "radical"],
+	185: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/9/1/4/914b66c2b145cb1d08b645c9ee80f8fb1023dc98.png", "Music", "radical"],
+	439: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/6/a/5/6a5bf551d2669cad828d27c42308888d7d652510.png", "Sell", "radical"],
+	187: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/3/e/7/3e75a5045502607b8c5dbc5a08245cf1e938979b.png", "Sick", "radical"],
+	200: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/1/7/9/17988fb04f7d4bc43a072be4bbc52d635ea222dc.png", "Stairs", "radical"],
+	179: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/1/3/1/1315e5857ace7fbabc95768dc524a4be47df7683.png", "Tombstone", "radical"],
+	240: ["https://aws1.discourse-cdn.com/wanikanicommunity/original/4X/c/7/d/c7d387bdbc6142d8c7bcd8059bb6c76e32812364.png", "Turkey", "radical"],
+};
+
+const channelIds = {
+	r: {
+		'1, 10': '1003903962690371645',
+		'11, 20': '1003903993493327892',
+		'21, 30': '1003904018482991174',
+		'31, 40': '1003904037575463013',
+		'41, 60': '1003904063211057224',
+	},
+	k: {
+		'1, 10' : '1003904128197599252',
+		'11, 20': '1003904166441259028',
+		'21, 30': '1003904204622004305',
+		'31, 40': '1003904271164649492',
+		'41, 50': '1003904297102225518',
+		'51, 60': '1003904321039126559',
+	},
+	v: {},
+};
+
+module.exports = {
+	async submitAll(client) {
+		const submit = require('./DBhandler.js').submit;
+		console.log(submit)
+
+		const getMeanings = meaning => meaning.split(', ').map(e => e.toLowerCase().replace(' ', ''));

```

Comment:
Rather than parse this in-place here I would do it in advance, at the time you create the radical.json, kanji.json, etc. Replace multiple meanings with a list. We want our cached json data to be structured so we don't have to parse it later at all.